packages/trader/src/Services/socket-general.js

```diff
@@ -1,12 +1,13 @@
-import { action, flow }  from 'mobx';
-import ObjectUtils       from 'deriv-shared/utils/object';
-import Login             from '_common/base/login';
-import ServerTime        from '_common/base/server_time';
-import BinarySocket      from '_common/base/socket_base';
-import { State }         from '_common/storage';
-import { localize }      from 'App/i18n';
-import { requestLogout } from './logout';
-import WS                from './ws-methods';
+import { action, flow }           from 'mobx';
+import ObjectUtils                from 'deriv-shared/utils/object';
+import Login,
+{ redirectToLogin }               from '_common/base/login';
+import ServerTime                 from '_common/base/server_time';
+import BinarySocket               from '_common/base/socket_base';
+import { State }                  from '_common/storage';
+import { localize }               from 'App/i18n';
+import { requestLogout }          from './logout';
+import WS                         from './ws-methods';
```

Comment:
too much white space between the import variable and the `from` :D

Reply:
You're right. I'm on it